packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
 import classnames from 'classnames';
-import {getAdaptiveCard} from '@ciscospark/react-component-utils';
+import {getAdaptiveCard, getDecryptedURls, API_ACTIVITY_VERB} from '@ciscospark/react-component-utils';
```

Comment:
Can you make this a relative import? we should avoid potential circular dependencies

Reply:
![Screenshot 2019-12-18 at 4 14 20 PM](https://user-images.githubusercontent.com/55378170/71079605-96bb3280-21b1-11ea-8074-303e1d7ab4ee.png)
While I'm running test cases. It's throwing an error as it's expecting in a particular format.